x-pack/plugins/logstash/server/routes/cluster/load.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { ICustomClusterClient, IRouter } from 'src/core/server';
+import { licenseCheckerRouteHandlerWrapper } from '../../../../licensing/server';
+import { Cluster } from '../../models/cluster';
+import { checkLicense } from '../../lib/check_license';
+
+export function registerClusterLoadRoute(router: IRouter, esClient: ICustomClusterClient) {
+  router.get(
+    {
+      path: '/api/logstash/cluster',
+      validate: false,
+    },
+    licenseCheckerRouteHandlerWrapper(
+      checkLicense,
+      router.handleLegacyErrors(async (context, request, response) => {
```

Comment:
Is the `router.handleLegacyErrors` necessary here as the whole handler body seems to be in a `try/catch` block returning proper NP errors?